invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/invitationsModal/InvitationsMembersModal.js

```diff
@@ -0,0 +1,225 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import {
+  Button,
+  Modal,
+  Tab,
+  Form,
+  Checkbox,
+  Item,
+} from "semantic-ui-react";
+import { i18next } from "@translations/invenio_communities/i18next";
+import _isEmpty from "lodash/isEmpty";
+import { withState } from "react-searchkit";
+import _upperFirst from "lodash/get";
+import { CommunityInvitationsApi } from "../../../api";
+import { ErrorMessage } from "../../components/ErrorMessage";
+import { SelectedMembers } from "./SelectedMembers";
+
+import { PeopleTabPane } from "./PeopleTabPane";
+import { GroupTabPane } from "./GroupTabPane";
+
+export class InvitationsMembersModal extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      open: false,
+      selectedMembers: [],
+      role: undefined,
+      message: "",
+      error: undefined,
+      isLoading: false,
+      suggestions: [],
+    };
+  }
+
+  inviteMembers = async () => {
+    const { communityID, updateQueryState, currentQueryState } = this.props;
+    const { selectedMembers, role, message } = this.state;
+    const client = new CommunityInvitationsApi();
+    const serializedMembers = selectedMembers.map((member) => {
+      return { type: member.type, id: member.id };
+    });
+    const payload = {
+      members: serializedMembers,
+      role: role,
+      message: message,
+    };
+    try {
+      this.setState({ isLoading: true });
+      await client.createInvite(communityID, payload);
+      // TODO: Remove the timeout once the check to see if it was indexed in ES is implemented in the backend
+      await new Promise((resolve) => setTimeout(resolve, 2000));
+      updateQueryState(currentQueryState);
+      this.setState({
+        open: false,
+        isLoading: false,
+        selectedMembers: [],
+        role: undefined,
+        message: "",
+        suggestions: [],
+        error: undefined,
+      });
+    } catch (e) {
+      this.setState({ error: e.response.data, isLoading: false });
+    }
+  };
+
+  roleOptions = () => {
+    const { roles } = this.props;
+    const { role } = this.state;
+
+    return Object.entries(roles).map(([key, value]) => (
+      <Item>
```

Comment:
The component is called `modal` but there is no modal inside it is a bit confusing